cars/v1/vanilla/config.ini

```diff
@@ -18,5 +18,5 @@ docker_image=docker.elastic.co/elasticsearch/elasticsearch
 # major version of the JDK that is used to build Elasticsearch
 build.jdk = 16
```

Comment:
Do we want to bump this as well? Java 17 will be required for 8.0, but I don't know enough about Java to tell if we should change `build.jdk`

Reply:
Oops, it's in #69